mlflow/server/js/src/utils/Utils.js

```diff
@@ -122,11 +125,11 @@ class Utils {
   static renderSource(run, tags) {
     let res = Utils.formatSource(run);
     if (run.source_type === "PROJECT") {
-      const match = run.source_name.match(Utils.getGitHubRegex());
-      if (match) {
-        let url = "https://github.com/" + match[1] + "/" + match[2].replace(/.git/, '');
-        if (match[3]) {
-          url = url + "/tree/master/" + match[3];
+      const GitHubMatch = run.source_name.match(Utils.getGitHubRegex());
+      if (GitHubMatch) {
+        let url = "https://github.com/" + GitHubMatch[1] + "/" + GitHubMatch[2].replace(/.git/, '');
+        if (GitHubMatch[3]) {
+          url = url + "/tree/master/" + GitHubMatch[3];
```

Comment:
Shouldn't this code be updated to support BitBucket too?

Reply:
Before I update that code,  I am facing this issue which I'm not able to solve. 

Line 232: static is a reserved word in strict mode

Kindly help. Thanks